custom_components/tuya_local/devices/LSC_SMART_Connect_RGB-CCT_Bulb

```diff
@@ -0,0 +1,106 @@
+name: LSC RGB-CCT
+products:
+  - id: hpaeslvkx4znwzae 
+    name: LSC SMART Connect A60 RGB-CCT
+
+primary_entity:
+  entity: light
+  dps:
+    - id: 1
+      type: boolean
+      name: switch
+    - id: 2
+      name: color_mode
+      type: string
+      mapping:
+        - dps_val: white
+          value: color_temp
+        - dps_val: colour
+          value: hs
```

Comment:
Add a mapping for `scene` here and the duplicate select entity below will not be needed.

Reply:
like the screenshot above, we have a scene selector, that works normal, and we have a work_mode selector that works also normal. If we add the scene as a mapping it won't work and fail, the scenes won't work. see screenshot below.




![image](https://github.com/make-all/tuya-local/assets/13116501/e97daee8-7bb0-4e8d-933a-414323148e29)
![image](https://github.com/make-all/tuya-local/assets/13116501/574505b8-4c0b-416f-b381-8fc16a3c7492)

Also i show you the updated code according your requests, and it make it looks bad.


```
name: Generic RGB-CCT Bulb
products:
  - id: hpaeslvkx4znwzae 
    name: LSC SMART Connect A60 RGB-CCT

primary_entity:
  entity: light
  dps:
    - id: 1
      type: boolean
      name: switch
    - id: 2
      name: color_mode
      type: string
      mapping:
        - dps_val: white
          value: color_temp
        - dps_val: colour
          value: hs
        - dps_val: scene
          value: scene
    - id: 3
      name: brightness
      type: integer
      range:
        min: 25
        max: 255
    - id: 4
      name: color_temp
      type: integer
      range:
        min: 0
        max: 255
      mapping:
        - target_range:
            min: 2700
            max: 6500
    - id: 5
      name: rgbhsv
      type: hex
      format:
        - name: r
          bytes: 1
        - name: g
          bytes: 1
        - name: b
          bytes: 1
        - name: h
          bytes: 2
          range:
            min: 0
            max: 360
        - name: s
          bytes: 1
          range:
            min: 0
            max: 255
        - name: v
          bytes: 1
          range:
            min: 0
            max: 255

secondary_entities:
  - entity: select
    name: Scene
    icon: "mdi:palette"
    category: config
    dps:
      - id: 6
        type: string
        name: option
        optional: true
        mapping:
          - dps_val: "ffff2801005766"
            value: Scene 1
          - dps_val: "ffff3d0266000003660000ff48000000000000000000"
            value: Scene 2
          - dps_val: "ffff2301096600"
            value: Scene 3
          - dps_val: "ffff0505ff000000ff00ffff00ff00ff0000ff000000"
            value: Scene 4
```

Reply:
Normally I would capitalise the non-standard entries in color_mode, as they do not have translations.
But it seems the HA change referenced in #1598 also came with a behaviour change that effects no longer auto cancel when you change modes, so implementation of that may need to be prioritised.